pkg/kedge/http/director/proxy.go

```diff
@@ -55,9 +57,15 @@ var (
 //
 // If Adhoc routing supports dialing to whitelisted DNS names either through DNS A or SRV records for undefined backends.
 func New(pool backendpool.Pool, router router.Router, adhocRouter adhoc.Addresser, logEntry logrus.FieldLogger) *Proxy {
+	ipMatchRegexp, err := regexp.Compile(`^\d+\.\d+\.\d+\.\d+$`)
+	if err != nil {
+		// This should never panic in production, only in unit tests.
```

Comment:
Please use that as package `var` and use `MustCompile`

Reply:
done